core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1920,8 +1930,15 @@ private DefaultServerConfig buildServerConfig(List<ServerPort> serverPorts) {
                 ChannelUtil.applyDefaultChannelOptions(
                         childChannelOptions, idleTimeoutMillis, pingIntervalMillis);
 
-        ServerErrorHandler errorHandler = this.errorHandler;
-        if (errorHandler != ServerErrorHandler.ofDefault()) {
+        ServerErrorHandler errorHandler;
+        if (!exceptionReportingInterval.isZero()) {
+            errorHandler = new ExceptionReportingServerErrorHandler(this.errorHandler,
+                                                                    exceptionReportingInterval.getSeconds());
+        } else {
+            errorHandler = this.errorHandler;
+        }
+
+        if (this.errorHandler != ServerErrorHandler.ofDefault()) {
```

Comment:
We have to do this before wrapping above.

Reply:
OK ! 👍 👍 👍